packages/react-components/src/components/VideoTile.tsx

```diff
@@ -240,16 +239,20 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
             background: theme.palette.neutralLighter,
             borderRadius: theme.effects.roundedCorner4
           },
+          isSpeaking && {
+            '&::before': {
+              content: `''`,
+              position: 'absolute',
+              zIndex: 1,
```

Comment:
should we make this a constant somewhere so we know why the z-index is being set to 1?

Reply:
Oops. I missed this comment. But I think it should be okay with a constant since the value is 1. If it was a number like 999 then I would.